testing/ostest/tls.c

```diff
@@ -48,17 +47,17 @@ static struct tls_info_s g_save_info;
 
 static void get_tls_info(FAR struct tls_info_s *info)
 {
-  memcpy(info, up_tls_info(), sizeof(struct tls_info_s));
+  memcpy(info, tls_get_info(), sizeof(struct tls_info_s));
```

Comment:
Maybe we can switch to define instead? Seems like a simple wrapper.

Reply:
Anyway minor

Reply:
Done.